src/components/App.jsx

```diff
@@ -21,7 +21,6 @@ import oauth from 'panoptes-client/lib/oauth';
 
 import { env } from '../config';
 import { WORKFLOW_STATUS } from '../ducks/workflow';
-import { SPLIT_STATUS } from '../ducks/splits';
```

Comment:
Can we remove `SPLIT_STATUS` from the duck? Doesn't look like it's being used anymore.